composer.json

```diff
@@ -72,6 +72,60 @@
     "bin": [
         "php-cs-fixer"
     ],
+    "scripts": {
+        "cs:check": "php php-cs-fixer fix --dry-run",
+        "cs:diff": "git diff HEAD --name-only --diff-filter=ACMR -- '*.php' | xargs -r php php-cs-fixer fix --dry-run --diff",
+        "cs:diff-fix": "git diff HEAD --name-only --diff-filter=ACMR -- '*.php' | xargs -r php php-cs-fixer fix",
+        "cs:fix": "php php-cs-fixer fix",
+        "dev-tools:check": [
+            "./dev-tools/check_file_permissions.sh",
+            "./dev-tools/check_trailing_spaces.sh"
```

Comment:
Instead of this I would like to introduce common `.editorconfig` that would ensure intendation, trailing new lines and spaces etc. WDYT? But in separate PR.

Reply:
editconfig is nice to help with your local development, but we need to check this on CI as well.

Reply:
OK. I wasn't able to run it properly on my machine, but Composer script itself works and invokes those scripts.

<img width="612" alt="image" src="https://user-images.githubusercontent.com/600668/225048311-e5a8fdd8-81e1-4cbd-9f99-7c9f83fb40b6.png">

`./dev-tools/check_file_permissions.sh` shouldn't return 0 exit code and it should fail, while `./dev-tools/check_trailing_spaces.sh` has false-positive on my computer, but this is out of the scope of this PR, so I'll leave it as is and it should be improved in separate PRs.